docs/fundamentals/code-analysis/style-rules/language-rules.md

```diff
@@ -37,12 +37,12 @@ Options for language rules can be specified in an EditorConfig file with the fol
 
 The style rules in this section are applicable to both C# and Visual Basic.
 
-- ["This." and "Me." qualifiers](#this-and-me)
+- ["This." and "Me." qualifiers](ide0003_ide0009.md)
```

Comment:
This table will still be retained in case user wants to browse all style rules by Option group and/or option names, without the overhead of rule IDs, which are only ever needed if user wants to change rule severity.